### ISSUE TITLE
add comments to import

### ISSUE THREAD

**Issue by hcientist** [JMU-CS/canvas-rubric-filler#1]
modify the script so that it would notice a column that holds comments for the given rubric item or submission as a whole and ensures those are entered. The post to canvas code [already looks for a "comment" item in the dictionary that we create from the csv file](https://github.com/JMU-CS/canvas-rubric-filler/blob/master/filler.py#L121), but in parsing the csv to create this dict we are not currently attempting to notice any comment columns 

so basically decide on a column header name convention (perhaps `<crit_id>_comments`?) and have the parsing code look for it and update the docs
